lib/mind_the_gapp_web/views/arrival_time_view.ex

```diff
@@ -0,0 +1,27 @@
+defmodule MindTheGappWeb.ArrivalTimeView do
+  use MindTheGappWeb, :view
+  use Timex
+
+  def render("index.json", %{data: data}) do
+    render_many(data, MindTheGappWeb.ArrivalTimeView, "one.json")
+  end
+
+  def render("one.json", %{arrival_time: [day, average | _rest]}) do
+    today = Timex.now
+    days_ago = calculate_difference_between_days(today, day)
+    day_label = ""
+    day_label = case days_ago do
+      0 ->
+        "Today"
+      1 ->
+        "Yesterday"
+      _ ->
+        "#{days_ago} Days Ago"
+    end
```

Comment:
Does it makes sense to extract this into a function with an intention revealing name?

Reply:
I think it could go either way. Do you think it would be more understandable if it was extracted to a function?

Reply:
I prefer to have smaller functions. I think a good litmus test is SRP. Does this function have a single reason to change?